Cargo.toml

```diff
@@ -2,6 +2,17 @@
 name = "fastexcel"
 version = "0.11.6"
 edition = "2021"
+include = [
+    "/pyproject.toml",
+    "/README.md",
+    "/LICENSE",
+    "/Makefile",
```

Comment:
needed ?

Reply:
yeah, otherwise everything gets included by default and we end up with a 50mb sdist :sweat_smile: